tsconfig-backend.json

```diff
@@ -1,9 +1,8 @@
 {
-  "extends": "@tsconfig/node14/tsconfig.json",
+  "extends": "@tsconfig/node16/tsconfig.json",
   "include": ["api/**/*", "api-lib/**/*", "scripts/**/*"],
   "compilerOptions": {
-    "lib": ["es2021", "esnext"],
     "resolveJsonModule": true,
-    "noErrorTruncation": true
```

Comment:
Hmm... I was motivated to do this because I was tinkering with upgrading typescript in a different branch, and that required updating Zeus, and at one point I got a typecheck error that dumped such a large object into the console, it somehow made me run out of hard drive space 😱 until I disabled unlimited scroll back in the terminal